core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.directive.html

```diff
@@ -75,50 +198,182 @@ <h2 class="dashboard-title" translate="I18N_DASHBOARD_TOPICS_AND_SKILLS_DASHBOAR
+    display: flex;
+    flex-wrap: wrap;
+    height: 100%;
+  }
+  .oppia-topics-and-skills-dashboard .create-topic-skill i {
+    color: #00645C;
+    font-size: 30px;
+    width: 100%;
+  }
+  .oppia-topics-and-skills-dashboard .create-topic-skill span {
+    color: #00645C;
+    display: inline-block;
+    font-size: 14px;
+    line-height: 15px;
+    width: 100%;
+  }
+  .oppia-topics-and-skills-dashboard .topic-skill-filter-content {
+    margin: 0 auto;
+    width: 90%;
+  }
+  .oppia-topics-and-skills-dashboard .topic-skill-filter-value {
+    width: 65%;
+  }
+  .oppia-topics-and-skills-dashboard .topic-skill-filter-value input,
+  .oppia-topics-and-skills-dashboard .topic-skill-filter-value select {
+    width: 100%;
+  }
+  .oppia-topics-and-skills-dashboard .classroom-dropdown .select2-results {
+    text-transform: capitalize;
+  }
+  .oppia-topics-and-skills-dashboard .topic-skill-filter-header {
+    margin: 10px 0 15px;
+    position: relative;
+  }
+  .oppia-topics-and-skills-dashboard .topic-skill-filter-heading {
+    font-size: 30px;
+    font-weight: bold;
+  }
+  .oppia-topics-and-skills-dashboard .topic-skill-filter-reset {
+    bottom: 0;
+    color: #00645C;
+    cursor: pointer;
+    position: absolute;
+    right: 0;
+  }
+  .oppia-topics-and-skills-dashboard .topic-skill-filter-row {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin: 8px 0;
+  }
+  .oppia-topics-and-skills-dashboard .topic-skill-filter-key {
+    display: inline-block;
+    text-align: right;
+    width: 25%;
+  }
+  .oppia-topics-and-skills-dashboard .topic-pagination-arrow {
+    cursor: pointer;
+  }
+  .oppia-topics-and-skills-dashboard .dashboard-list-header {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin: 30px 0 50px;
+  }
+  .oppia-topics-and-skills-dashboard .dashboard-list-pagination {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    width: 30%;
+  }
+  .oppia-topics-and-skills-dashboard .dashboard-list-pagination-value {
+    cursor: pointer;
+    padding: 0 4px;
+  }
+  .oppia-topics-and-skills-dashboard .dashboard-list-pagination-active {
+    color: #00645C;
+  }
   .oppia-topics-and-skills-dashboard .dashboard-tabs {
-    display: -webkit-flex;
     display: flex;
+    display: -webkit-flex;
     flex-wrap: wrap;
-    margin-bottom: 0;
-    margin-left: 12.5%;
-    padding-left: 0;
-    padding-top: 6%;
+    margin: 0;
+    padding: 0;
     text-align: center;
   }
-
   .oppia-topics-and-skills-dashboard .create-buttons {
     display: none;
     margin-left: 40vw;
   }
-
   .oppia-topics-and-skills-dashboard .dashboard-tabs li {
-    display: -webkit-flex;
     display: flex;
+    display: -webkit-flex;
     margin-bottom: 0;
-    width: 250px;
+    width: 50%;
   }
   .oppia-topics-and-skills-dashboard .dashboard-title {
     color: #01645c;
     font-family: 'Capriola', 'Roboto', Arial, sans-serif;
```

Comment:
I think quotes are not required here, can you remove those?

Reply:
Done.